website/docs/r/serviceendpoint_nexus.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_serviceendpoint_nexus"
+description: |-
+  Manages a Service Connection for Nexus IQ.
+---
+
+# azuredevops_serviceendpoint_nexus
+
+Manages a Nexus IQ service endpoint within Azure DevOps, which can be used as a resource in YAML pipelines to connect to a Nexus IQ instance.
```

Comment:
[Nexus IQ](https://marketplace.visualstudio.com/items?itemName=SonatypeIntegrations.nexus-iq-azure-extension) is not supported by default,   we should document that to manage a nexus service connection resource,  users need to install the nexus extension first.